kqueen_ui/blueprints/manager/templates/manager/overview.html

```diff
@@ -122,29 +122,45 @@ <h3>
 
 {% block extrajs %}
 <script>
+var handleError = function (elId, errorDetails) {
```

Comment:
Why don't you show flash message instead?

Reply:
it is shown as part of kqueen request exception handling. I'm saving error info to show it to the user on a hover to a failed table